lib/Interpreter/IncrementalParser.h

```diff
@@ -175,6 +175,14 @@ namespace cling {
       return m_Transactions.back();
     }
 
+    ///\brief Merge transactions after the one given.
+    ///
+    ///\param[in] T - transactions after which to merge
+    ///\param[in] prev - merge into transaction prior to T or T itself
+    ///\returns the parent transaction of the merge.
+    ///
+    const Transaction* mergeTransactionsAfter(const Transaction *T, bool prev);
```

Comment:
1. To use this functionality I don't think a requirement should be an `#include "Transaction.h"` and the caller knowing how Transactions are implemented.

2. The difference is more complicated than T or T->getNext().  As **Previous** get's the Transaction before what was passed.